tox.ini

```diff
@@ -86,9 +86,8 @@ deps =
   codecov
 commands =
   /bin/sh {toxinidir}/scripts/create_test_db_user.sh {env:DATABASE_USER} {env:DATABASE_ADMIN}
-  pipenv run pip install pip==18.0
   pipenv install --dev  --ignore-pipfile
-  coverage run --parallel-mode --concurrency=multiprocessing {toxinidir}/koku/manage.py test --noinput --parallel -v 2 {posargs: koku/masu/}
+  coverage run --parallel-mode --concurrency=multiprocessing {toxinidir}/koku/manage.py test --noinput --parallel -v 2 {posargs: masu.test.database.test_aws_report_db_accessor.ReportDBAccessorTest.test_populate_line_item_daily_summary_table_no_bill_ids}
```

Comment:
Forgot to undo this file

Reply:
I specifically staged a highlighted area to make sure this DIDN'T happen, and it still did. <ugh>